### DISCUSSION TITLE
Get Podfile from build

### DISCUSSION THREAD

**Discussion by Xeladrane91** [codemagic-ci-cd/codemagic-docs#1135] [Q&A] [answered]
Hello,
Is there a way to retrieve the Podfile generated from a ios build ? Or get the whole project ios built ?
Kind regards

**Comment by icarusdust** [comment, marked as answer]
Hi, you can create the Podfile manually in your project as well and use the generated content. Or push the changes directly from a builder machine. Check [this documentation](https://docs.codemagic.io/troubleshooting/accessing-builder-machine-via-ssh/#making-configuration-changes).

**Reply by Xeladrane91** [reply]
Thank you for your answer 👍
